compiler/testData/debug/stepping/conjunction.kt

```diff
@@ -0,0 +1,24 @@
+//FILE: test.kt
+fun box() {
+    val k = if (getA()
+        && getB()
+        && getC()
+        && getD()) {
+        true
+    } else {
+        false
+    }
+}
+
+fun getA() = true
+
+fun getB() = true
+
+fun getC() = false
+
+fun getD() = true
+
+// LINENUMBER: TestKt.box():3 TestKt.getA():13 TestKt.box():3 TestKt.box():4 TestKt.getB():15 TestKt.box():4 TestKt.box():5
```

Comment:
Sorry for delay.
I would prefer to have tests in next format:
// LINENUMBERS
// TestKt.box():3 
// TestKt.getA():13
// TestKt.box():3 
// TestKt.box():4 
// TestKt.getB():15 
// TestKt.box():4 
// TestKt.box():5
...

Reply:
Done!